cmd/cmd_pkg.go

```diff
@@ -230,6 +229,24 @@ func loadCommandHandler(cmd *cobra.Command, args []string) {
 		exitWithError(err.Error())
 	}
 
+	// Validate local package
+	packageFolder := filepath.Base(pkgFlags.PackagePath())
```

Comment:
we're really the only ones generating packages today so while this is ok we're mainly concerned with ensuring that one isn't trying to run a macho-o